servicetalk-grpc-protoc/src/main/java/io/servicetalk/grpc/protoc/Generator.java

```diff
@@ -597,7 +598,17 @@ private void extractJavaDocComments(State state, int methodIndex, MethodSpec.Bui
                 .addStatement("return this")
                 .build());
 
-        final MethodSpec.Builder addBlockingServiceMethodSpecBuilder = methodBuilder(addService)
+        serviceBuilderSpecBuilder.addMethod(methodBuilder(addService)
+                .addModifiers(PUBLIC)
+                .addAnnotation(Deprecated.class)
+                .addJavadoc(JAVADOC_DEPRECATED + "Use {@link #$L($T)}." + lineSeparator(), addBlockingService,
```

Comment:
We have to add a full javadoc. Otherwise, there are other warnings, like missing `@param` that break the build with `Xwerror` flag.

Reply:
will submit a followup

Reply:
this is what I meant about javadoc generation not reliably detecting issues. hopefully https://github.com/apple/servicetalk/pull/2140 will help but there are protobuf issues.

Reply:
https://github.com/apple/servicetalk/pull/2144